### DISCUSSION TITLE
`Uncaught ReferenceError: require is not defined` error when trying to use `'next-auth/providers/linkedin'`

### DISCUSSION THREAD

**Discussion by hirehamir** [nextauthjs/next-auth#8614] [Help]
I have `NEXTAUTH_URL` set to the domain of my branch's Vercel preview deployment, with the starting `https://`, but I get an ` Uncaught ReferenceError: require is not defined at 9sy73bkb829663kxb6g04uafc:1:14` message in the browser console and a `Bummer, something went wrong.` message from LinkedIn.

```
// 9sy73bkb829663kxb6g04uafc
var _jsecure = require("@linkedin/jsecure")
  , _jsecure2 = _interopRequireDefault(_jsecure);
function _interopRequireDefault(a) {
    return a && a.__esModule ? a : {
        default: a
    }
}
window.setTimeout(function() {
    _jsecure2.default.redirect(document.getElementById("redirect-uri").textContent)
}, 5E3);
```

![image](https://github.com/nextauthjs/next-auth/assets/43425812/9ffb51d7-3814-4bbf-9317-01a896429a36)

**Comment by ShrijalFetchly** [comment]
Is it fixed? :')
